CONTRIBUTING.md

```diff
@@ -36,7 +36,7 @@ Target //:jazzer_release up-to-date:
 
 #### macOS
 
-The build may fail with the clang shipped with Xcode. 
+The build may fail with the clang shipped with Xcode.
 In that case, install LLVM clang via `brew install llvm` and set `CC` to the path of LLVM clang.
 
```

Comment:
This requirement is going away with https://github.com/CodeIntelligenceTesting/jazzer/pull/557. Since Jazzer ships libFuzzer, we can use Xcode instead of Homebrew LLVM. That PR deletes the `macOS` section entirely.

Reply:
Neat! Please add me as a reviewer there as well once it's done.
This would also mean cleaner docs and guides for cifuzz.

Reply:
I will, but I don't think this will affect cifuzz: It simplifies building Jazzer from source, which isn't something cifuzz does. As far as I know Xcode doesn't ship libFuzzer, so *using* cifuzz for C++ projects on macOS still requires LLVM clang.